content/testing-farm/_index.md

```diff
@@ -0,0 +1,246 @@
+        - fedora-30-x86_64
+        - fedora-rawhide-x86_64
+
+    - job: tests
+      trigger: pull_request
+      metadata:
+        targets:
+        - fedora-29-x86_64
+        - fedora-30-x86_64
+        - fedora-rawhide-x86_64
+
+That's it! Since now you should see testing feedback in your pull requests.
+
+### Support Matrix
+
+Currently supported steps and implementations:
+
+* Discover: fmf
+* Prepare: ansible
+* Execute: shell, beakerlib
+
+### Test Environment
+
+We currently support running tests in an x86_64 virtual machine only. The system is using qemu-kvm with hardware acceleration. You have full root access in the machine. The test environment currently cannot be easily changed.
+
+Additional machine specs:
+
+* RAM: 2 GB
+* HDD: 4 GB
+
+### Debugging and Reproducing
+
+We are working on an easy way how to develop new tests, debug and reproduce issues for failed runs.
+Until that is ready, your best option is to spin up a new Fedora Docker container or run the Fedora Cloud Base qcow2 manually via qemu-kvm or libvirt.
+See the [Worker](#worker) section for details how to run the tool locally.
+
+
+## Examples
+
+Get inspiration for a quick start from a couple of real-life examples!
+
+### Shell Tests
+
+Here is an example of a simple integration test for the web server `httpd` and `curl` utility:
+
+    /test/build/smoke:
+        execute:
+            how: shell
+            commands:
+            - dnf -y install httpd curl
+            - systemctl start httpd
+            - echo foo > /var/www/html/index.html
+            - curl http://localhost/ | grep foo
+
+Single testset `/test/build/smoke` defines only the `execute` step.
+Individual shell commands are provided as a list.
+Testing will fail if any of the commands returns a non-zero exit status.
+
+### BeakerLib Tests
+
+Below you can find little bit more interesting example of a `systemd` test configuration:
+
+    /test/pull-request/smoke:
+        summary:
+            Basic set of quick smoke tests for systemd.
+        discover:
+            how: fmf
+            filter: "tier: 1 & distros: rhel-8"
+            repository: "https://github.com/systemd-rhel/tests"
+        prepare:
+            how: ansible
+            playbooks: [ci/rhel-8.yml]
+        execute:
+            how: beakerlib
+
+This testset enables a set of Tier 1 tests from the shared [systemd tests][systemd-tests] repository.
+The meaning of individual attributes is as follows:
+
+* Summary — an optional but useful attribute describing high-level purpose of the testset.
+* Discover — instructs to fetch tests from given repository and select relevant ones by provided `filter`.
+* Prepare — specifies which ansible playbook should be applied to prepare environment for testing.
+* Execute — defines that the beakerlib framework should be used for running the tests.
+
+## Issues & RFEs
+
+If you have found an issue or have an RFE, please use Packit Service's Github issue tracker to [file an issue][issues].
```

Comment:
We should probably discuss this. Don't you want to have your own issue tracker?

Reply:
I'm ok with tracking testing farm issues in a separate project. But this is rather question for @thrix.

Reply:
changed it